workshops/interactive_maps/README.md

```diff
@@ -0,0 +1,279 @@
+---
+name: 'Interactive Maps'
+description: 'Learn JS and HTML while making a fun interactive map'
+author: '@LonaDotExe'
```

Comment:
The author name should be same as your github username. And in that case it is JeswinSunsi right?